### PR TITLE
Flow example

### PR DESCRIPTION
**Pull request by BenjiLee**
@lxu213 This is a flow example.

The benefits for flow here is gradual migration is much easier. We don't have to create separate `tsx` files, we only need to add `//@flow` at the top of files to get type checking. 

The double edged sword here is that we get the benefit of being able to run the code as it is today with flow errors, but that means that we need to manually check for types. Typescript forces all issues to be resolved before it can be compiled.

### CODE REVIEW

File: src/utils/constants.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 // number of sim curves to be displayed in Graph
-export const numDisplaySims = 30;  
+export const numDisplaySims = 30;
 
 // id: explicit key to list items in onChange()
 // key: use this to key into dataset
 // name: display name for visualization
-export const STATS = [
+
+export type Stat = {
```
Comment (BenjiLee) [inline src/utils/constants.js:10]: Not that I didn't have to create a completely separate file like in https://github.com/HopkinsIDD/covid-dashboard-app/pull/86/files#diff-dc8854f26b91c0ebbb07727882dbf479R5

File: src/components/Filters/Indicators.js [hunk 2]
```diff
@@ -1,8 +1,25 @@
+// @flow
```
Comment (BenjiLee) [inline src/components/Filters/Indicators.js:1]: Note that I didn't need to fix any other file's reference to this one like in Typescript https://github.com/HopkinsIDD/covid-dashboard-app/pull/86/files#diff-e20d0897545286867d50af9026addf07R7

File: src/components/Filters/Indicators.js [hunk 3]
```diff
@@ -31,7 +48,7 @@ class Indicators extends Component {
         this.setState({children})
     }
 
-    handleChange = (e) => {
+    handleChange = (e: any) => {
```
Comment (BenjiLee) [inline src/components/Filters/Indicators.js:51]: Note that `antd` does not have a built flowtype for this. It uses the TypeScript https://github.com/HopkinsIDD/covid-dashboard-app/pull/86/files#diff-ecd2affe445bc1a8b13c4648a0bf3550R4


That being said,  the flow community attempts to add types for popular libraries so it is possible to get types for most popular project. It requires pulling down a flow-typed file https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/antd_v3.x.x/flow_v0.104.x-/antd_v3.x.x.js

File: src/components/Filters/Indicators.js (hunk 2 above)
Comment (lxu213) [reply src/components/Filters/Indicators.js:1]: That would make migration much more seamless, for sure.

**Pull request closed without merging**